apidoc/Makefile

```diff
@@ -1,11 +1,35 @@
-html: clean
-	cp -r zinc .working
-	cp LibZincDoxygen/api_2_api++_doc.py .working
-	cp -r LibZincDoxygen/libzinc_doxgen_script .working
-	cd .working; python api_2_api++_doc.py
-	cd .working/libzinc_doxgen_script; doxygen cpp_comments.doxygen
+html: clean zincapi ironapi
+
+zincapi:
+# Copies the OpenCMISS-Iron source code to a working directory then run a custom script, and then doxygen to generate the relevant API documentation.
```

Comment:
This comment mentions Iron not Zinc. It's worth updating both the comments for Zinc and Iron to briefly state what they actually do; Zinc runs a script to transfer doxygen from its C to C++ API, then generates the C++ API docs. Iron has a few different steps.

Reply:
Hi @rchristie , thanks I have updated the comments to better describe what's going on. Let me know if you're happy with that.